tasks/lib/layouts_manager.js

```diff
@@ -17,6 +17,14 @@ var defaultLayouts = {
 
   byComponent: handleUntyped(function(type, pkg) {
     return path.join(pkg, type);
+  }),
+
+  byTypeKeepStructure: handleUntyped(function(type, pkg, sourceDir) {
```

Comment:
I'm not familiar with this syntax ` byTypeKeepStructure: handleUntyped(function`. Can you explain it please?